### PR TITLE
[jsk_perception][jsk_recognition_utils] support chainer-v2 in alexnet and vgg16

### PR DESCRIPTION
**Pull request by knorth55**
- `alexnet` and `vgg16` support chainer-v2
- fix syntax in `vgg16_object_recognition.py`
- enable `alexnet` and `vgg` test

### CODE REVIEW

File: jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/vgg16_batch_normalization.py [hunk 1]
```diff
@@ -42,45 +42,42 @@ def __init__(self, n_class=1000):
...
-        h = F.dropout(F.relu(self.fc7(h)), train=self.train, ratio=0.5)
+        h = F.dropout(F.relu(self.fc6(h)), ratio=0.5)
+        h = F.dropout(F.relu(self.fc7(h)), ratio=0.5)
         h = self.fc8(h)
         fc8 = h
 
         self.pred = F.softmax(h)
 
         if t is None:
-            assert not self.train
```
Comment (wkentaro) [inline jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/vgg16_batch_normalization.py]: `assert not chainer.config.train`

File: jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/vgg16.py [hunk 2]
```diff
@@ -55,13 +54,12 @@ def __call__(self, x, t=None):
...
 
-        h = F.dropout(F.relu(self.fc6(h)), train=self.train, ratio=0.5)
-        h = F.dropout(F.relu(self.fc7(h)), train=self.train, ratio=0.5)
+        h = F.dropout(F.relu(self.fc6(h)), ratio=0.5)
+        h = F.dropout(F.relu(self.fc7(h)), ratio=0.5)
         h = self.fc8(h)
 
         self.pred = F.softmax(h)
         if t is None:
-            assert not self.train
```
Comment (wkentaro) [inline jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/vgg16.py]: ditto.

File: jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/alexnet_batch_normalization.py [hunk 3]
```diff
@@ -16,23 +16,21 @@ def __init__(self, n_class=1000):
...
         h = F.max_pooling_2d(F.relu(self.conv5(h)), 3, stride=2)
-        h = F.dropout(F.relu(self.fc6(h)), train=self.train)
-        h = F.dropout(F.relu(self.fc7(h)), train=self.train)
+        h = F.dropout(F.relu(self.fc6(h)))
+        h = F.dropout(F.relu(self.fc7(h)))
         h = self.fc8(h)
 
         self.pred = F.softrmax(h)
         if t is None:
-            assert not self.train
```
Comment (wkentaro) [inline jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/alexnet_batch_normalization.py]: ditto.

File: jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/vgg16_batch_normalization.py (hunk 1 above)
Comment (knorth55) [inline jsk_recognition_utils/python/jsk_recognition_utils/chainermodels/vgg16_batch_normalization.py]: ok

**Comment by knorth55** [comment]
updated

**Pull request merged**